README.md

```diff
@@ -300,7 +300,7 @@ docker run
     --network     host                                                           \
     --hostname    signing-service                                                \
     --name        signing-service                                                \
-    --volume      contract.key:/etc/contract.key:ro                              \
+    --volume      < path to file >/contract.key:/etc/contract.key:ro             \
```

Comment:
I don't think this change is necessary. `contract.key` is a relative path and it's explained below that it's just one of the possible locations:

> - There's a file called `contract.key` in your local directory and that it contains the private key that can sign Ethereum transactions from the account owned by Concent contract.